internal/proxy/meta_cache.go

```diff
@@ -478,7 +479,7 @@ func (m *MetaCache) describeCollection(ctx context.Context, collectionName strin
 		CreatedUtcTimestamp:  coll.CreatedUtcTimestamp,
 	}
 	for _, field := range coll.Schema.Fields {
-		if field.FieldID >= common.StartOfUserFieldID {
+		if field.FieldID >= common.StartOfUserFieldID || field.IsDynamic {
```

Comment:
Since the fieldID of the dynamic field is greater than common.StartOfUserFieldID, is the second criterion unnecessary?